plugins/inventory/cobbler.py

```diff
@@ -44,6 +44,13 @@
         type: list
         default: []
         elements: str
+        version_added: 4.4.0
+      include_profiles:
+        description: Profiles to include from inventory (exclude all others)
```

Comment:
```suggestion
        description:
          - Profiles to include from inventory.
          - If specified, all other profiles will be excluded.
          - I(exclude_profiles) is ignored if I(include_profiles) is specified.
```
The documentation for `exclude_profiles` should say something similar about `include_profiles`.